src/frontend/src/stream_fragmenter/mod.rs

```diff
@@ -294,6 +294,7 @@ fn build_fragment(
         }
 
         NodeBody::StreamCdcScan(_) => {
+            // XXX: Should we use a different flag for CDC scan?
```

Comment:
Depending on whether we find the need to distinguish between them.

Reply:
I think we do need to distinguish them. See the awkward code in `tracking_progress_actor_ids`. But I'm worried about compatibility so don't want to change it.